README.md

```diff
@@ -17,39 +147,49 @@ git clone https://github.com/sparkletown/sparkle
 cd sparkle
 ```
 
-In the main `sparkle` folder, copy `.env.example` to a new file `.env.local`, and fill it with the appropriate details.
-
-You can read more about the various `.env` files that you can use at:
-
-- https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env
+### Step 2: Install Packages
 
 Install the platform dependencies with `npm`:
 
-(**Note:** `npm` v7+ is not supported, it will cause issues with our `package-lock.json`, and you may end up with the wrong dependency versions)
+**Important Note:** `npm` v7+ is not supported, it will cause issues with our `package-lock.json`, and you may end up with the wrong dependency versions. Check your current npm version with `npm -v`.
 
 ```bash
 npm install
 ```
 
+### Step 3: Connect with Firebase
+
+In the main `sparkle` folder, copy `.env.example` to a new file `.env.local`, and fill it with the appropriate details, most of which you can find in the Firebase dashboard.
+
+You can read more about the various `.env` files that you can use at: https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env
+
+
+### Step 4: Start Your Engines
+
 Now you're ready to start the server! ✨
 
 ```bash
 npm start
 ```
 
-Once the server is started, your web browser will be opened at http://localhost:3000 (and then it'll be immediately redirected to https://sparklespaces.com/). To start using the app, navigate to a URL such as http://localhost:3000/v/{venueName} - replacing `{venueName}` with the existing venue you'd like to use.
+Once the server is started, your web browser will be opened at http://localhost:3000 (and then it'll be immediately redirected to https://sparklespaces.com/).
+
+To start using the app, you must have a venue setup. Venue URLs take the form of http://localhost:3000/v/{venueName} - replacing `{venueName}` with the existing venue you'd like to use.
+
+If this is your first time running Sparkle and you are using a fresh Firebase, we have simplified the initial venue setup and admin access using scripts found in the **/scripts/** directory.
```

Comment:
should we mention here exact steps to follow? (create a bootstrap venue, add themselves admins, etc)
or at least, tell where these instructions can be found.